Branch-SDK/src/io/branch/referral/BranchConfig.java

```diff
@@ -0,0 +1,8 @@
+package io.branch.referral;
+
+/*
+ *
+ */
+public class BranchConfig {
```

Comment:
My bad. I created BNCConfig on iOS side, but too lazy to do the same on Android side at that time.
Anyway, I'd prefer moving all the stuff to here in one shot. People getting this release will find this class with so little content. That doesn't look good :-)